mkroosevelt.js

```diff
@@ -45,7 +51,14 @@ else if (cmd === '-v' || cmd === '--v' || cmd === '-version' || cmd === '--versi
 }
 else if (cmd) {
   try {
-    createSampleApp();
+    if (cmd === '.') {
+      currentDirectory = true;
+      createSampleApp(currentDirectory);
```

Comment:
Doesn't seem to work correctly. I created a fresh app in the same directory, altered app.js, then reran `mkroosevelt .` in that app and it overwrote my changes.